Digipost.Signature.Api.Client.Direct/DirectClient.cs

```diff
@@ -50,6 +49,12 @@ public async Task<JobResponse> Create(Job job)
             return directJobResponse;
         }
 
+        public async Task<SignerResponse> RequestNewRedirectUrl(SignerResponse signerResponse)
```

Comment:
Jeg er litt todelt på denne. Det er lettest å sende inn en URI hit, men av erfaring så kødder avsendere med slikt. Jeg tenker det derfor er fint å se at det er responsen man bruker for å kunne hente en ny URL, og at man, hey, også får en slik tilbake.

Reply:
Fikset med `IWithSignerUrl`